modules/apps/headless/headless-delivery/headless-delivery-impl/src/main/java/com/liferay/headless/delivery/internal/resource/v1_0/DocumentDataDefinitionTypeResourceImpl.java

```diff
@@ -34,16 +40,64 @@
  * @author Javier Gamarra
  */
 @Component(
-	properties = "OSGI-INF/liferay/rest/v1_0/document-data-definition-types.properties",
+	properties = "OSGI-INF/liferay/rest/v1_0/document-data-definition-type.properties",
 	scope = ServiceScope.PROTOTYPE,
-	service = DocumentDataDefinitionTypesResource.class
+	service = DocumentDataDefinitionTypeResource.class
 )
-public class DocumentDataDefinitionTypesResourceImpl
-	extends BaseDocumentDataDefinitionTypesResourceImpl {
+public class DocumentDataDefinitionTypeResourceImpl
+	extends BaseDocumentDataDefinitionTypeResourceImpl {
+
+	@Override
+	public Page<DocumentDataDefinitionType>
+			getAssetLibraryDocumentDataDefinitionTypesPage(Long assetLibraryId)
+		throws Exception {
+
+		return _getDocumentDataDefinitionTypesPage(
+			HashMapBuilder.put(
+				"createBatch",
```

Comment:
Does it make sense to have also [create](https://github.com/liferay/liferay-portal/blob/48d2ff809e9ff838012d6dd5fd48c19dc069cad6/modules/apps/headless/headless-delivery/headless-delivery-impl/src/main/java/com/liferay/headless/delivery/internal/resource/v1_0/DocumentResourceImpl.java#L177) and [get](https://github.com/liferay/liferay-portal/blob/48d2ff809e9ff838012d6dd5fd48c19dc069cad6/modules/apps/headless/headless-delivery/headless-delivery-impl/src/main/java/com/liferay/headless/delivery/internal/resource/v1_0/DocumentResourceImpl.java#L192) ?

Reply:
Also, does it make sense to have actions per item [here](https://github.com/liferay/liferay-portal/blob/2d163697d7a240a67561afcdaa7b5c1cccb036dc/modules/apps/headless/headless-delivery/headless-delivery-impl/src/main/java/com/liferay/headless/delivery/internal/resource/v1_0/DocumentDataDefinitionTypeResourceImpl.java#L209)? maybe only the get like https://github.com/liferay/liferay-portal/blob/48d2ff809e9ff838012d6dd5fd48c19dc069cad6/modules/apps/headless/headless-delivery/headless-delivery-impl/src/main/java/com/liferay/headless/delivery/internal/resource/v1_0/DocumentResourceImpl.java#L974

Reply:
Done